lib/temperature/simulator.js

```diff
@@ -0,0 +1,44 @@
+/* -*- mode: js; js-indent-level:2; -
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2019-present Samsung Electronics France and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var console = require('console');
+
+function Simulator () {
+  this.value = {
+    temperature: 0,
```

Comment:
maybe just keep value as number , no need to add structure at this level

See API of actual driver

Reply:
can you link to said api so I can look ?

Reply:
forget what I said your structure is good:
https://github.com/rzr/generic-sensors-lite/blob/master/lib/temperature/index.js#L61

For reference
https://github.com/dbridges/bmp085-sensor/blob/master/bmp085-sensor.js